packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -28,6 +28,105 @@ Email templates use common and unique [Expression Language (EL) variables](https
+This EA feature supports the use of functions within email templates, but how you call the function's syntax is different. For example, instead of calling functions in the EL templating syntax like this:
+
+`${f:formatTimeDiffHoursNowInUserLocale(org.activationTokenExpirationHours)}`
+
+You would call functions in the Velocity Templating syntax like this:
+
+`$f.formatTimeDiffHoursNowInUserLocale($org.activationTokenExpirationHours)`
+
+#### Access to all User Profile attributes
+
+You can now reference any attribute in a user's Okta User Profile in your email templates. The notation is `$user.profile.attributeName`. For example, to reference the displayName attribute of the user's Okta user profile, you would use `$user.profile.displayName`.
+
+See [Profile object](/docs/api/resources/users#profile-object) for more information on the available user profile attributes.
+
+##### Example of user attributes
+
+###### Registration: Email verification with customer's preferred language
+Trigger an email in your end-users' preferred language by using the conditional logic and calling the `preferredLanguage` attribute.
+
+###### Activation department specific email
+If a customer wants to specify a department such as Engineering in an Activation Email, they are able to call the user attribute `department`.
+
+#### Access to org attributes
+
+You can also reference some org-level attributes:
+
+* `$org.name`
+* `$org.locale`
+* `$org.subDomain`
+
+#### Support for conditional logic
+
+All conditional logic that is supported by the Velocity Templating Engine, such as `if`, `elseif`, or `else` constructs and `foreach` loops, is available for you to use in your templates. See the [Velocity documentation](http://velocity.apache.org/engine/1.7/user-guide.html).
+
+#### List of Customizable Templates
+
+The following email templates are available for customization:
+
+* Account Lockout
+* Email Verification
+* Change Email Confirmation
+* Email Challenge
+* MFA Factor Enrolled
+* MFA Factor Reset
+* Forgot Password
+* Forgot Password Denied
+* Active Directory Password Reset
+* Active Directory Password Reset Denied
+* LDAP Forgot Password
+* Send Push Verify Activation Link
+* Registration - Activation
+* Registration - Email Verification
+* New Device Notification
+* Password Reset by Admin
+* Self-Service Unlock Account
+* Active Directory Self-Service Unlock Account
+* Self-Service Unlock when Account is not Locked
+* User Activation
+* Active Directory User Activation
+* LDAP User Activation
+
 ### Test Custom Email Templates
 
 You can send yourself a test email to see how a custom email template looks and functions. This can help you validate macro attributes and translations in the customized template as well as see how the template renders in different email environments. This eliminates the need to create a real end-to-end workflow to test customization. The test email is sent to the primary email address of the admin that initiates the test email.
```

Comment:
Should you call out the button to click?

Reply:
The procedure below this paragraph (step 2) calls out the **Send test email** button.